skins/oasis/modules/BodyController.class.php

```diff
@@ -111,12 +111,17 @@ public static function showUserPagesHeader() {
 		// perform namespace and special page check
 		$isUserPage = in_array( $wg->Title->getNamespace(), self::getUserPagesNamespaces() );
 
-		$ret = ( $isUserPage && !$wg->Title->isSubpage() )
-			|| $wg->Title->isSpecial( 'Following' )
-			|| $wg->Title->isSpecial( 'Contributions' )
-			|| $wg->Title->isSpecial( 'UserActivity' )
-			|| ( defined( 'NS_BLOG_LISTING' ) && $wg->Title->getNamespace() == NS_BLOG_LISTING )
-			|| ( defined( 'NS_BLOG_ARTICLE' ) && $wg->Title->getNamespace() == NS_BLOG_ARTICLE );
+		$ret =
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$ret". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)